pages/faq.js

```diff
@@ -1,8 +1,8 @@
 import Head from 'components/head';
 import HeroBanner from 'components/HeroBanner/HeroBanner';
 import Content from 'components/Content/Content';
-import FAQItem from 'components/FAQ/FAQItem/FAQItem';
-import QuestionAnswerData from 'components/FAQ/questions';
+import AccordionItem from 'components/Accordion/AccordionItem/AccordionItem';
+import QuestionAnswerData from 'components/Accordion/questions';
```

Comment:
Seems like `components/Accordion/questions.js` isn't the right spot for the file since it's not relavant to the Accordion component - just the `/faq` page...

Let's just bring the content of that file into this file. We can't make a separate file for it and put it into `pages` because every `.js` file in the `pages` folder becomes a route (that's how Next.js works). I don't want it to be in some random folder just for the sake of a confusing modularization attempt.

Reply:
So I moved the `questions.js` into the `Accordion` component and I'm getting a ` Only one default export allowed per module.` error. I basically just copied and pasted one component into the other.  I'm still a React noob - I apologize for the silly questions!

Reply:
Ok so I moved the `questions.js` file into the `Accordion` component (basically just copy & paste) and changed `export default ...` to just `export` and then the respective component name since I was getting a `Only one default export allowed per module` error. I'm now getting a `Support for the experimental syntax 'exportDefaultFrom' isn't currently enabled` error. It's suggesting `Add @babel/plugin-proposal-export-default-from (https://git.io/vb4yH) to the 'plugins' section of your Babel config to enable transformation.` but I'm hesitant to add any plug ins on my own since this is part of a much larger project